tests/page-objects/default/enketo/death-report.page.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment');
+const genericForm = require('./generic-form.wdio.page');
+
+const PLACE_OF_DEATH = {healthFacility: 'health_facility', home: 'home', other: 'other'};
```

Comment:
should the properties be uppercase since refer to constants? What are your thoughts? I'm not 100% strong about it

Reply:
After reading your comment I started looking to see if there was like a rule or some standards related to this, I found a comment in a forum that talks a little bit about that:

> ... Uppercase indicates that the variable is a constant and they are immutable.
But the properties inside a constant are mutable and hence they CAN be declared as lowercase.

And checking the examples in [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#const_in_objects_and_arrays) it looks like it is ok to leave the properties in lower case 🙂

Reply:
Nice, thanks for checking, let's leaving like this